README.md

```diff
@@ -4,6 +4,8 @@
 
 This is the repository for the UQ Design System.
 
+The UQ Design System is a guide to how user interfaces should look, feel, behave and be coded at UQ.  It includes a guide to the core styles, components, patterns and standards for UQ user interfaces.
```

Comment:
When I read this it seems like both sentences are saying pretty much the same thing. Firstly, in layman's terms and then in a more jargony fashion. I reckon the second sentence is redundant.